### PR TITLE
Enhance csv export to accept CollectionRepresentation

### PR DESCRIPTION
**Pull request by luca-rath**
| Q | A
| --- | ---
| Bug fix? | no
| New feature? | no
| BC breaks? | no
| Deprecations? | no
| Fixed tickets | fixes #issuenum
| Related issues/PRs | #issuenum
| License | MIT
| Documentation PR | sulu/sulu-docs#prnum

#### What's in this PR?

Enhance csv export to accept `PaginatedRepresentation`

#### Why?

Currently, csv export is only available for `ListRepresentation`, but it would work for `PaginatedRepresentation` too.

### CODE REVIEW

File: src/Sulu/Component/Rest/Csv/CsvHandler.php [hunk 1]
```diff
@@ -70,7 +70,7 @@ public function __construct(ArraySerializerInterface $serializer)
      */
     public function createResponse(ViewHandler $handler, View $view, Request $request, $format)
     {
-        if (!$view->getData() instanceof ListRepresentation) {
+        if (!$view->getData() instanceof PaginatedRepresentation) {
```
Comment (nnatter) [inline src/Sulu/Component/Rest/Csv/CsvHandler.php:73]: i think we could even use `CollectionRepresentation` here, right? 🙂

**Review by alexander-schranz — APPROVED** [review]

**Comment by alexander-schranz** [comment]
@luca-rath Looks good for me would bei create to have a unit test for it.

**Comment by danrot** [comment]
@nnatter Since when do we simply ignore comments and merge? :see_no_evil:

**Comment by nnatter** [comment]
To me, testing this did not feel necessary and @alexander-schranz already approved the PR too. But added the test for you here: https://github.com/sulu/sulu/pull/5237

**Comment by nnatter** [comment]
@danrot To me, testing this did not feel necessary. Then I saw that @alexander-schranz already approved the PR anyway and merged it. But added the test for you here: https://github.com/sulu/sulu/pull/5237

**Comment by danrot** [comment]
It looks to me like @alexander-schranz made that approval with one minor comment... And I think testing is especially important for stuff like that, because you seem to need that for an external project, and Sulu itself does not care. So it would be easily broken in the future again, if we don't test it.

**Comment by luca-rath** [comment]
well, thank you for writing the test for me 😉

**Pull request merged**